src/main/resources/skills.graphqls

```diff
@@ -20,33 +20,60 @@ type SkillDTO {
   examples: String
   category: Category!
   grade: Grade!
-  craftIds: [String!]
+  craftTitles: [String!]
```

Comment:
Should this be `[CraftTitle!]!`?

Reply:
yes have updated (and it threw an error because i have mobile(iOS) in the underlying data so fixed that). good spot, thanks!